go/pkg/rexec/rexec.go

```diff
@@ -415,6 +415,32 @@ func (ec *Context) GetOutputFileDigests(useAbsPath bool) (map[string]digest.Dige
 	return res, nil
 }
 
+// GetOutputDigests returns a map of output files to digests and a map of output directories to digests.
```

Comment:
Can you expand the comments of this vs GetOutputFileDigests saying that they're different because the latter will walk down the directories?